.gitignore

```diff
@@ -71,3 +71,22 @@ plugins/plugin_directory.xml
 # vscode
 .vscode
 *.code-workspace
+
+# Custom
+STACEY_NOTES.md
+./OUT/*
+OUT
```

Comment:
Adding OUT is probably okay, but the rest of the additions in this file are kind of specific and probably shouldn't be in the pull request.

Reply:
I had added OUT to the .gitignore but it looks like it was still added to the commit. I will take another look.

Reply:
STACEY_NOTES.md should also have been in the .gitignore. Apologies for not catching this before pushing.